content/en/docs/tasks/tools/install-kubectl-windows.md

```diff
@@ -66,6 +66,11 @@ The following methods exist for installing kubectl on Windows:
    ```cmd
    kubectl version --client
    ```
+   Or use this for detailed view of version:
+
+   ```cmd
+   kubectl version --client --output=yaml
+   ```
```

Comment:
In my view: if we make this change, we should make the equivalent change to the Windows and Mac OS docs too.

Reply:
It should be linux and mac OSinstead. Since I have changed in Windows.

Reply:
Ah, yep.

Reply:
Pages:
- https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/
- https://kubernetes.io/docs/tasks/tools/install-kubectl-macos/
- https://kubernetes.io/docs/tasks/tools/install-kubectl-windows/